### PR TITLE
fixed template according to the comments

### PR DESCRIPTION
**Pull request by anetta999**

### CODE REVIEW

File: src/pages/FirstSessionPage/FirstSessionPage.tsx
Comment (lyudad) [inline src/pages/FirstSessionPage/FirstSessionPage.tsx:1]: why this page even needed?

File: src/pages/PrivacyPolicyPage/PrivacyPolicyPage.tsx
Comment (lyudad) [inline src/pages/PrivacyPolicyPage/PrivacyPolicyPage.tsx:1]: rename to index.tsx

File: src/pages/RestorePasswordPage/RestorePasswordPage.tsx
Comment (lyudad) [inline src/pages/RestorePasswordPage/RestorePasswordPage.tsx:1]: index.tsx

File: src/styles/GlobalStyles.ts
Comment (lyudad) [inline src/styles/GlobalStyles.ts:1]: why name start from upper case

File: src/pages/FirstSessionPage/FirstSessionPage.tsx
Comment (anetta999) [reply src/pages/FirstSessionPage/FirstSessionPage.tsx:1]: I deleted FirstSessionPage in this commit https://github.com/ZenBit-Tech/faq_team_fe/pull/1/commits/620a019523e922324e006c45c027fccb745f8299

File: src/pages/PrivacyPolicyPage/PrivacyPolicyPage.tsx
Comment (anetta999) [reply src/pages/PrivacyPolicyPage/PrivacyPolicyPage.tsx:1]: renamed all files inside the folder of a component to index.tsx, styles.ts in this commit https://github.com/ZenBit-Tech/faq_team_fe/pull/1/commits/620a019523e922324e006c45c027fccb745f8299

File: src/pages/RestorePasswordPage/RestorePasswordPage.tsx
Comment (anetta999) [reply src/pages/RestorePasswordPage/RestorePasswordPage.tsx:1]: renamed all files inside the folder of a component to index.tsx, styles.ts in this commit https://github.com/ZenBit-Tech/faq_team_fe/pull/1/commits/620a019523e922324e006c45c027fccb745f8299

File: src/styles/GlobalStyles.ts
Comment (anetta999) [reply src/styles/GlobalStyles.ts:1]: renamed GlobalStyles to globalStyles and Theme to theme inside styles folder in this commit https://github.com/ZenBit-Tech/faq_team_fe/pull/1/commits/620a019523e922324e006c45c027fccb745f8299

File: src/components/InputField/styles.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  padding: 0px;
+  gap: 8px;
+  width: 335px;
+  height: 72px;
+`;
+
+export const Label = styled.label`
+  font-size: 16px;
+  color: #000;
+`;
```
Comment (lyudad) [inline src/components/InputField/styles.ts:18]: 1.3.2

File: src/components/InputField/styles.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  width: 100%;
+  padding: 8px;
+  border-radius: 8px;
+  border-width: 1px;
+  border: solid;
+  border-color: ${props => (props.hasError ? 'red' : '#ccc')};
+  outline: none;
+
+  &:focus {
+    border-color: #000;
```
Comment (lyudad) [inline src/components/InputField/styles.ts]: 1.3.2

File: src/types/index.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+export type User = {
+  name: string;
+  email: string;
+  role: string;
+};
+
+export type AuthState = {
+  user: User;
+  token: string;
+};
```
Comment (lyudad) [inline src/types/index.ts]: I'm not sure when you will use this file, if for redux, better to move types to redux, its will be hard to support this file over time

File: src/components/InputField/styles.ts (hunk 1 above)
Comment (anetta999) [reply src/components/InputField/styles.ts:18]: fixed colors to props.theme.colors.black in this commit https://github.com/ZenBit-Tech/faq_team_fe/pull/1/commits/6946b1fe06bc34499ad0193d2c5396d51a17d410

File: src/components/InputField/styles.ts (hunk 2 above)
Comment (anetta999) [reply src/components/InputField/styles.ts]: fixed colors to props.theme.colors.black in this commit https://github.com/ZenBit-Tech/faq_team_fe/pull/1/commits/6946b1fe06bc34499ad0193d2c5396d51a17d410

File: src/types/index.ts (hunk 3 above)
Comment (anetta999) [reply src/types/index.ts]: moved types to redux in this commit https://github.com/ZenBit-Tech/faq_team_fe/pull/1/commits/6946b1fe06bc34499ad0193d2c5396d51a17d410

File: src/components/InputField/styles.ts [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0px;
+  gap: 8px;
+  width: 335px;
+  height: 72px;
+`;
+
+export const Label = styled.label<{ theme?: Theme }>`
+  font-size: 16px;
```
Comment (lyudad) [inline src/components/InputField/styles.ts]: 1.3.2

**Pull request merged**